lib/demos/reference/motion_demo_container_transition.dart

```diff
@@ -0,0 +1,540 @@
+            ),
+            child: Text(
+              'Lorem ipsum dolor sit amet, consectetur '
+              'adipiscing elit, sed do eiusmod tempor.',
+              style: Theme.of(context)
+                  .textTheme
+                  .bodyText2
+                  .copyWith(color: Colors.black54, inherit: false),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _SmallerCard extends StatelessWidget {
+  const _SmallerCard({
+    this.openContainer,
+    this.subtitle,
+  });
+
+  final VoidCallback openContainer;
+  final String subtitle;
+
+  @override
+  Widget build(BuildContext context) {
+    return _InkWellOverlay(
+      openContainer: openContainer,
+      height: 225,
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: <Widget>[
+          Container(
+            color: Colors.black38,
+            height: 150,
+            child: Center(
+              child: Image.asset(
+                'assets/demos/motion/placeholder_image.png',
+                width: 80,
+              ),
+            ),
+          ),
+          Expanded(
+            child: Padding(
+              padding: const EdgeInsets.all(10.0),
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: <Widget>[
+                  Text(
+                    GalleryLocalizations.of(context).demoMotionPlaceholderTitle,
+                    style: Theme.of(context).textTheme.headline6,
+                  ),
+                  const SizedBox(height: 4),
+                  Text(
+                    subtitle,
+                    style: Theme.of(context).textTheme.caption,
+                  ),
+                ],
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _ExampleSingleTile extends StatelessWidget {
+  const _ExampleSingleTile({this.openContainer});
+
+  final VoidCallback openContainer;
+
+  @override
+  Widget build(BuildContext context) {
+    const height = 100.0;
+
+    return _InkWellOverlay(
+      openContainer: openContainer,
+      height: height,
+      child: Row(
+        children: <Widget>[
+          Container(
+            color: Colors.black38,
+            height: height,
+            width: height,
+            child: Center(
+              child: Image.asset(
+                'assets/demos/motion/placeholder_image.png',
+                width: 60,
+              ),
+            ),
+          ),
+          Expanded(
+            child: Padding(
+              padding: const EdgeInsets.all(20.0),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: <Widget>[
+                  Text(
+                    GalleryLocalizations.of(context).demoMotionPlaceholderTitle,
+                    style: Theme.of(context).textTheme.subtitle1,
```

Comment:
can do `final textTheme = Theme.of(context).textTheme at the top of the build to reuse here

Reply:
done